src/webgpu/api/validation/render_pass/storeOp.spec.ts

```diff
@@ -1,13 +1,156 @@
-export const description = `API Validation Tests for RenderPass StoreOp.
-
-  Test Coverage Needed:
-
-  - Test that when depthReadOnly is true, depthStoreOp must be 'store'
-
-  - Test that when stencilReadOnly is true, stencilStoreOp must be 'store'`;
+export const description = `
+API Validation Tests for RenderPass StoreOp.
+`;
 
 import { makeTestGroup } from '../../../../common/framework/test_group.js';
 
 import { ValidationTest } from './../validation_test.js';
 
 export const g = makeTestGroup(ValidationTest);
+
+g.test('depthStoreOp_must_be_store_when_depthReadOnly_is_true').fn(async t => {
```

Comment:
Rather than having both the control case and failure case expanded out like this, we can use parameterization to condense it. In addition, we can combine all of these tests into one.

Opened a PR with these suggestions here:
https://github.com/bjjones/cts/pull/1
(untested)
and I think it covers the comments I have below too.

Reply:
Using your code now, which resolves everything else.